lua/plugins/treesitter.lua

```diff
@@ -3,7 +3,16 @@ return {
   dependencies = {
     "JoosepAlviste/nvim-ts-context-commentstring",
     "nvim-treesitter/nvim-treesitter-textobjects",
-    "windwp/nvim-ts-autotag",
+    -- FIXME: remove when https://github.com/windwp/nvim-ts-autotag/issues/125 closed.
```

Comment:
```suggestion
    -- HACK: remove when https://github.com/windwp/nvim-ts-autotag/issues/125 closed.
```

I tend to use `FIXME` when there is a known issue/edge case or something incomplete. I think `HACK` is a better annotation here